sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/write/db/InsertQueryBuilder.java

```diff
@@ -106,6 +107,9 @@ public Query bind(
     values.addValues(row.getDoubleValue() == null ? Values.NULL : Values.of(row.getDoubleValue()));
     values.addValues(convertToBackendBooleanValue(row.getBooleanValue(), numericBooleans));
 
+    if (ttl != null) {
+      values.addValues(Values.of(ttl));
+    }
```

Comment:
did not know that `ttl` can also be the param.. was this possible in V1 as well? Thanks for the fix..

Reply:
The v1 implementation actually turns it into a bind variable under the hood. It also supports mixing such implicit markers with explicit ones well, but in v2 that would be a bit more complicated. So it's simpler to be explicit everywhere.